new/LIT7301QeneBalelit.xml

```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7301QeneBalelit" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በሌሊት፡ ለሞዓብ፡ አመ፡ ያጠፍኣ፡ ወአመ፡ ተካፈሉ፡ ሕዝብ፡ በርበረ፡ ቤተ፡ ሞዓብ፡ ኵሎ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ba-lelit la-Moʿāb ʾama yāṭaffǝʾā wa-ʾama takāfalu ḥǝzb barbara 
+                    beta Moʿāb kʷǝllo (qǝne)</title>
+                <title xml:lang="en" corresp="#t1">In the night, when he was destroying Moab, and when the people got separated, he 
+                    ransacked the entire house of Moab...</title>
```

Comment:
the story is probably the standard one from 2Kings https://www.biblica.com/bible/?osis=niv:2%20Kings%203